next.config.js

```diff
@@ -33,6 +33,14 @@ const moduleExports = {
     forceSwcTransforms: true,
   },
   headers: () => headers(),
+  async rewrites() {
+    return [
+      {
+        source: '/api/s3-upload/:path*',
+        destination: `https://${process.env.S3_BUCKET_NAME}.s3.eu-west-2.amazonaws.com/:path*`,
+      },
+    ]
+  },
```

Comment:
This is the only way I could upload files directly to S3 without getting a CORS error.